src/extension.ts

```diff
@@ -2,10 +2,10 @@
  * Copyright (C) Microsoft Corporation. All rights reserved.
  *--------------------------------------------------------*/
 
+import { tmpdir } from 'os';
```

Comment:
There are some re-ordered imports because the sorting imports codeaction is enabled by default on this repo